apps/condo/domains/common/components/AudioPlayer.tsx

```diff
@@ -0,0 +1,155 @@
+import styled from '@emotion/styled'
+import { Col, Row, Select } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import React, { CSSProperties, useEffect, useRef, useState } from 'react'
+import WaveSurfer from 'wavesurfer.js'
+
+import { Pause, Play } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { Typography } from '@open-condo/ui'
+import { colors } from '@open-condo/ui/dist/colors'
+
+const { Option } = Select
+
+const StyledSelect = styled(Select)`
```

Comment:
Why we need custom here? We have Select in UI-kit

Reply:
removed, used default select here